provisionersdk/scripts/bootstrap_windows.ps1

```diff
@@ -35,6 +35,19 @@ if (-not (Get-Command 'Set-MpPreference' -ErrorAction SilentlyContinue)) {
 $env:CODER_AGENT_AUTH = "${AUTH_TYPE}"
 $env:CODER_AGENT_URL = "${ACCESS_URL}"
 
+$psi = [System.Diagnostics.ProcessStartInfo]::new("$env:TEMP\sshd.exe", '--version')
+$psi.UseShellExecute = $false
+$psi.RedirectStandardOutput = $true
+$p = [System.Diagnostics.Process]::Start($psi)
+$output = $p.StandardOutput.ReadToEnd()
+$p.WaitForExit()
+
+if ($output -notlike "*Coder*") {
```

Comment:
At least ps has some nice features, even if it looks monstrous 😄

Reply:
Can't spell "Powershell" without "hell" 💀